app/views/admin/tags/index.html.erb

```diff
@@ -1,27 +1,29 @@
 <div class="flex items-center mb-6">
-  <ul class="nav nav-pills">
-    <li class="nav-item">
-      <a href="/admin/tags" class="nav-link <%= "active" if params[:state].blank? %>">All</a>
-    </li>
-    <li class="nav-item">
-      <a href="/admin/tags?state=supported" class="nav-link <%= "active" if params[:state] == "supported" %>">Supported</a>
-    </li>
-    <li class="nav-item">
-      <a href="/admin/tags?state=unsupported" class="nav-link <%= "active" if params[:state] == "unsupported" %>">Unsupported</a>
-    </li>
-  </ul>
+  <div class="crayons-tabs">
+    <div class="nav-item">
+      <a href="/admin/tags" class="crayons-tabs__item <%= "crayons-tabs__item--current" if params[:state].blank? %>">All</a>
+    </div>
+    <div class="nav-item">
+      <a href="/admin/tags?state=supported" class="crayons-tabs__item <%= "crayons-tabs__item--current" if params[:state] == "supported" %>">Supported</a>
+    </div>
+    <div class="nav-item">
+      <a href="/admin/tags?state=unsupported" class="crayons-tabs__item <%= "crayons-tabs__item--current" if params[:state] == "unsupported" %>">Unsupported</a>
+    </div>
+  </div>
 
-  <div class="ml-auto">
-    <%= form_tag(admin_tags_path, method: "get") do %>
-      <div class="form-group">
-        <%= text_field_tag(:search, params[:search], aria: { label: "Search" }) %>
-        <% if params[:state].present? %>
-          <%= hidden_field_tag(:state, params[:state]) %>
-        <% end %>
-        <%= submit_tag("Search") %>
-      </div>
+  <div class="m-auto">
+    <%= form_tag(admin_tags_path, method: "get", class: "inline-flex") do %>
+      <%= text_field_tag(:search, params[:search], aria: { label: "Search" }, class: "crayons-textfield") %>
+      <% if params[:state].present? %>
+        <%= hidden_field_tag(:state, params[:state]) %>
+      <% end %>
+      <%= submit_tag("Search", class: "crayons-btn ml-2") %>
     <% end %>
   </div>
+
+  <div class="justify-content-end ml-auto">
+    <%= link_to "Make A Tag", new_admin_tag_path, class: "crayons-btn" %>
```

Comment:
what about "Create" ? It sounds like "make a wish" to me :P

Reply:
True, I was following other admin pages that use this terminology. Maybe those other ones are better suited for "Make" like "Make a New Broadcast".

I'll switch this up to "Create a new Tag"

Reply:
oh, sorry I didn't realize that. Let's keep it as it is then :D